Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -973,6 +973,7 @@
             "com.apple.replaykit.sharingsession"
             "com.apple.tccd.system"
             "com.apple.replayd"
+            "com.apple.windowserver.active"
```

Comment:
It would be great if we could resolve this issue without allowing this sandbox extension. Is there another approach we could take? If this is a regression, perhaps we could track down the regression point?

Reply:
Done with API @eric-carlson mentionned